docs/standard/containerized-lifecycle-architecture/design-develop-containerized-apps/visual-studio-tools-for-docker.md

```diff
@@ -3,60 +3,51 @@ title: Using Visual Studio Tools for Docker (Visual Studio on Windows)
+The developer workflow when using Visual Studio Tools for Docker is similar to the workflow when using Visual Studio Code and Docker CLI (in fact, it is based on the same Docker CLI), but it is easier to get started, simplifies the process, and provides greater productivity for the build, run, and compose tasks. It's also able to execute and debug your containers via simple actions like F5 and Ctrl+F5 from Visual Studio. With the optional Container Orchestration support, in addition to being able to run and debug a single container, you also can run and debug a group of containers (a whole solution) at the same time if they are defined in the same *docker-compose.yml* file at the solution level.
 
 ## Configuring your local environment
 
 With the latest versions of Docker for Windows, it is easier than ever to develop Docker applications because the setup is straightforward, as explained in the following references.
 
 **More info:** To learn more about installing Docker for Windows, go to <https://docs.docker.com/docker-for-windows/>.
 
-If you're using Visual Studio 2015, you must have Update 3 or a later version plus the Visual Studio Tools for Docker.
-
-**More info:** For instructions on installing Visual Studio, go to [https://visualstudio.microsoft.com/\
-products/vs-2015-product-editions](https://visualstudio.microsoft.com/products/vs-2015-product-editions).
+**More info:** For instructions on installing Visual Studio, go to [https://visualstudio.microsoft.com/downloads/](https://visualstudio.microsoft.com/downloads/).
 
 To see more about installing Visual Studio Tools for Docker, go to <http://aka.ms/vstoolsfordocker> and <https://docs.microsoft.com/aspnet/core/host-and-deploy/docker/visual-studio-tools-for-docker>.
 
-If you're using Visual Studio 2017, Docker support is already included.
-
-## Using Docker Tools in Visual Studio 2015
-
-The Visual Studio Tools for Docker provides a consistent way to develop and validate locally your Docker containers for Linux in a Linux Docker host or VM, or your Windows Containers directly on Windows.
-
-If you're using a single container, the first thing you need to begin is to turn on Docker support into your .NET Core project. To do this, right-click your project file, as shown in Figure 4-25.
-
-![https://i1.visualstudiogallery.msdn.s-msft.com/0f5b2caa-ea00-41c8-b8a2-058c7da0b3e4/image/file/205468/1/add-docker-support.png](./media/image31.png)
-
-Figure 4-25: Turning on Docker support for your Visual Studio project
+Docker support is included in Visual Studio 2017.
```

Comment:
Specify the VS workload that needs to be installed to get the Docker tooling. Also consider mentioning the Docker for Windows requirement.